src/libs/actions/BankAccounts.ts

```diff
@@ -154,7 +155,7 @@ function connectBankAccountWithPlaid(bankAccountID: number, selectedPlaidBankAcc
  *
  * TODO: offline pattern for this command will have to be added later once the pattern B design doc is complete
  */
-function addPersonalBankAccount(account: PlaidBankAccount) {
+function addPersonalBankAccount(account: PlaidBankAccount, bankAccountList: BankAccountList) {
```

Comment:
There should be only one default account. Let's make this function accept only the default bank account

Reply:
Sorry, but I didn't understand, what do you mean by "accept only the default bank account" in "addPersonalBankAccount", this function is not used to add new accounts only?

Reply:
> There should be only one default account

I wanted to modify the `optimisticData: [` to add the default tag to the newly added bank, always ensuring "only one default account", however we only have the `BankAccountID` after receiving the response from the backend. Do you have any better ideas? Thanks.

Reply:
Instead of passing `bankAccountList` just pass `currentDefaultBankAccount`